code/game/gamemodes/blob/blobs/heatsink.dm

```diff
@@ -0,0 +1,24 @@
+/obj/effect/blob/heatsink
+	name = "heatsink blob"
+	icon_state = "blank_blob"
+	desc = "A fleshy mass capable of asborbing laser fire."
```

Comment:
>asborbing

I'd suggest `A pulsating mass supported by thin tendrils, capable of absorbing laser fire.`